migrations/0012_auto_20200727_0859.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-07-27 08:59
+from __future__ import unicode_literals
+
+import django.core.files.storage
+from django.db import migrations, models
+import django.db.models.deletion
+import plugins.books.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('books', '0011_auto_20190731_1456'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Category',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255)),
+                ('slug', models.SlugField(max_length=255)),
+            ],
+        ),
+        migrations.AlterField(
+            model_name='book',
+            name='cover',
+            field=models.FileField(blank=True, null=True, storage=django.core.files.storage.FileSystemStorage(location='/vol/janeway/src/media'), upload_to=plugins.books.models.cover_images_upload_path),
```

Comment:
We should use `JanewayFileSystemStorage` here to avoid the harcoded media path

Reply:
SORTED